packages/deployer/subtasks/prepare-deployment.js

```diff
@@ -38,10 +34,7 @@ subtask(
 
   mkdirp.sync(hre.deployer.paths.instance);
 
-  const { previousFile, currentFile } = await _determineDeploymentFiles(
-    hre.deployer.paths.instance,
-    alias
-  );
+  const { previousFile, currentFile } = await _determineDeploymentFiles(alias);
```

Comment:
The `_determineDeploymentFiles` function takes 2 params instead of 1: `(instance, alias)`

Reply:
Fixed